psi4/src/export_mints.cc

```diff
@@ -691,6 +693,23 @@ void export_mints(py::module& m) {
              "Returns a new Matrix object named name with default dimensions");
 //              py::arg("name"), py::arg("symmetry"));
 
+    py::class_<CdSalc::Component, std::shared_ptr<CdSalc::Component>>(
+        m, "SalcComponent", "Component of a Cartesian displacement SALC in the basis of atomic displacements.")
+        .def_readwrite("coef", &CdSalc::Component::coef, "The coefficient of the displacement")
+        .def_readwrite("atom", &CdSalc::Component::atom, "The index of the atom being displaced")
+        .def_readwrite("xyz", &CdSalc::Component::xyz, "The direction of tthe displacement, given by x as 1, y as 2, z as 3.");
```

Comment:
For `atom`, whenever there's atom indexing mentioned in user-facing docs, please add "0-indexed" or "1-indexed". For `xyz`, typo in "tthe" and is it really 1-indexed?

Thanks for the docstrings.